src/libstd/process.rs

```diff
@@ -47,6 +47,14 @@ use thread::{self, JoinHandle};
 ///
 /// assert!(ecode.success());
 /// ```
+///
+/// # Safety
```

Comment:
AFAIK the "Safety" heading is for memory safety and memory safety only. Also AFAIK a child process terminating later than expected is not a memory safety hazard. If I'm correct about both things, this heading should be renamed.